Project2/src/main/java/com/revature/Project2/beans/Swimlane.java

```diff
@@ -37,11 +37,24 @@
 	@JoinColumn(name="S_ID")
 	@OrderColumn
 	private List<Story> stories;
+	
+	@Column(name="B_ID")
```

Comment:
this does not alter the structure of the database, it just creates a field on the java side so you can perform java logic on it.